splink/comparison_library.py

```diff
@@ -487,3 +487,58 @@ def create_description(self) -> str:
 
     def create_output_column_name(self) -> str:
         return self.col_expression.output_column_name
+
+
+class DistanceInKMAtThresholds(ComparisonCreator):
+    def __init__(
+        self,
+        lat_col: str,
+        long_col: str,
+        km_thresholds=[0.1, 1],
```

Comment:
Appreciate this is probably to be consistent with Splink 3 implementation, but I think there probably shouldn't be a default here.  I don't think there are values which make sense for 'most users ' (unlike e.g. jaro winkler where I I think defaults make more sense)

Reply:
Ah yeah, good point, hadn't really given it much thought when I copied bits across.